README.md

```diff
@@ -180,6 +180,15 @@ Please use `<BLANKLINE>` when an output contains blank lines.
  */
 def helloWorld = "Hello\n\nWorld"
 ```
+
+## Ignoring Some Files
+
+If you don't want to generate doctests for some of your sources, then specify a regex pattern:
+
+```scala
+doctestIgnorePattern := Some(".*SomeClass.scala")
```

Comment:
I prefer the word `Regex` to `Pattern` to avoid confusion of kind of pattern (regular expression or glob pattern).